Source/Core/Core/HW/BBA-TAP/TAP_Win32.cpp

```diff
@@ -304,15 +304,17 @@ bool CEXIETHERNET::RecvStart()
 	DWORD res = ReadFile(mHAdapter, mRecvBuffer, BBA_RECV_SIZE,
 		(LPDWORD)&mRecvBufferLength, &mReadOverlapped);
 
-	if (!res && (GetLastError() != ERROR_IO_PENDING))
+	DWORD err = GetLastError();
+	if (!res && err != ERROR_IO_PENDING)
 	{
 		// error occurred
+		ERROR_LOG(SP1, "Failed to recieve packet with error 0x%X", err);
 		return false;
 	}
 
 	if (res)
```

Comment:
I mean like:
```c++
	if (res) {
		// Since the read is synchronous here, complete immediately
		RecvHandlePacket();
		return true;
	} else {
		DWORD err = GetLastError();
		if (err == ERROR_IO_PENDING) {
			// Will be completed asynchronously
			return true;
		}
		// Unexpected error
		ERROR_LOG(SP1, "Failed to recieve packet with error 0x%X", err);
		return false;
	}
```